### ISSUE TITLE
torchmetrics Accuracy

### ISSUE THREAD

**Issue by Antoine101** [Lightning-AI/pytorch-lightning#19885]
### 📚 Documentation

Hi Lightning team,

I was looking at metrics and found something a tad confusing.

In the following screenshot, the accuracy metric is imported like so:
`torchmetrics.classification.Accuracy`
![image](https://github.com/Lightning-AI/pytorch-lightning/assets/48209504/b574fcb4-bc84-4703-a3cb-9e9a71d16422)

But looking at this other page of the Torchmetrics doc, it shows: `torchmetrics.Accuracy` (without `classification` in between)
![image](https://github.com/Lightning-AI/pytorch-lightning/assets/48209504/1bd392dc-431b-4afb-a9f5-6e06620b40ed)

In my script, the following work:
`import torchmetrics.Accuracy`
`import torchmetrics.classification.accuracy`

But not the following:
`import torchmetrics.classification.Accuracy`

Which one is the right one?

If it is `torchmetrics.Accuracy`, why do the other classification metrics seem to require the `classification` in between?
![image](https://github.com/Lightning-AI/pytorch-lightning/assets/48209504/ede854fa-bcb7-469c-be39-6b9e85b4c693)

Cheers

Antoine

**Comment by Borda** [comment]
@Antoine101 thank you for sharing your convenes, just to clarify, this is related only to docs? and what version are you referring to?

**Comment by Borda** [comment]
> But yeah doc related.

so no issue with importing metrics both ways:
- `torchmetrics.Accuracy`
- `torchmetrics.classification.Accuracy`

**Comment by Antoine101** [comment]
torchmetrics.Accuracy ✅
torchmetrics.classification.accuracy ✅
torchmetrics.classification.Accuracy ❌

And so what is the difference between the two first ones?
